libs/platform/src/lib/components/search-field/search-field.component.ts

```diff
@@ -215,6 +226,7 @@ export class SearchFieldComponent extends BaseComponent implements OnInit, OnDes
     public submitId = '';
     public menuId = '';
     public dir = 'ltr';
+    currentSearchSuggestionAnnoucementMessage = '';
```

Comment:
Should that field have any access modifier? And if it's not a part of public API it should starts with `_`

Reply:
now it is not getting used in template, made it private and included `_` in variable name.